app/assets/javascripts/pages/agent-edit-page.js.coffee

```diff
@@ -108,10 +108,12 @@ class @AgentEditPage
     $(".control-link-region").show()
 
   hideEventCreation: ->
-    $(".event-related-region").hide()
```

Comment:
Found it https://github.com/DatabaseCleaner/database_cleaner#rspec-with-capybara-example